README.md

```diff
@@ -2,7 +2,7 @@
 
 Realm is a mobile database that runs directly inside phones, tablets or wearables.
 
-This repository holds adapters for combining Realm Java with Android UI components and framework classes.
+This repository holds adapters for combining the Realm Android SDK with Android UI components and framework classes.
```

Comment:
I would suggest linking to realm-java.
But we haven't renamed realm-java to "Realm Android"?

Reply:
Good point on linking to the SDK library -- I think a lot of users who stumble upon this page will appreciate the link.

As for the naming... in the docs, we use the term "Android SDK" to more easily distinguish between realm-swift and realm-java -- the thought is, it's easier for users to find the SDK they care about if they can focus on the platform. It also helps clarify that realm-java only works with Android apps, not arbitrary java applications.

However... those names aren't set in stone. Especially with the Kotlin Multiplatform (or Kotlin Native?) SDK coming up, it might be better to rethink that naming convention, since Android apps will presumably just start using the Kotlin SDK at some point.

I think I'd be more comfortable wording this as "realm-java (also known as the Realm Android SDK)" given our naming convention in the docs. But we're probably long overdue some active thought on the naming convention in the docs... if you feel strongly that we should consider renaming "Android SDK" realm-java, I can make that happen pretty quickly and we can just rephrase this to "Realm Java SDK" or "realm-java" or "Realm Java".

@bmunkholm let me know what you think, great question.

Reply:
```suggestion
This repository holds adapters for combining [realm-java](https://github.com/realm/realm-java) (the Realm Android SDK) with Android UI components and framework classes.
```